app-web-frontend/src/Home/HealthProviderHome.js

```diff
@@ -0,0 +1,13 @@
+import Header from '../header/Header';
+
+
+function HealthProviderHome() {
+    return (
+        <div className="App">
```

Comment:
Currently we add Header component here. We'll rework it afterwards, but while we didn't properly setup Routing it should appear here